doc/0.123/_sources/reference/get-campaign-views-budget.rst.txt

```diff
@@ -0,0 +1,46 @@
+.. index:: GET /campaign/{id}/views/budget
+.. _get_campaign_views/budget:
+
+GET /campaign/{id}/views/budget
+===============================
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.ad-lite.list-v1+json``
+
+Returns an overview of ``adLite`` objects, based on matching query parameters, in the context of a particular campaign. In addition, it returns the total size of the
+result set matching the parameters criteria, and a next page token if additional pages are available.
+This view should be used to quickly get insight on the ads in a campaign that might need attention or optimizing, from an ad total budget perspective. It should not be used for scrapping the entire inventory of ads from that campaign.
```

Comment:
```suggestion
This view should be used to quickly get insight into the ads in a campaign that might need attention or optimizing, from an ad total budget perspective. It should not be used for scrapping the entire inventory of ads from that campaign.
```

Reply:
this is completely unrelated to oauth2, and we'll likely revisit these APIs anyways

Reply:
indeed, but all those generated txt/pdf files have nothing to do with my change :)